metrics/storage/fio_job.sh

```diff
@@ -238,6 +256,9 @@ function main()
 	docker stop ${CONTAINER_ID}
 	docker rm ${CONTAINER_ID}
 
+	# Drop the host page cache
+	sudo bash -c "echo 3>/proc/sys/vm/drop_caches"
```

Comment:
I thought we'd be dropping the host caches between the `FIO_PREP_JOB` and the `FIO_RUN_JOB`, not after we have completed the test? Give me your thoughts here?